supertokens_python/recipe/session/interfaces.py

```diff
@@ -78,16 +111,59 @@ async def create_new_session(
     ) -> SessionContainer:
         pass
 
```

Comment:
It's difficult in python because python only provides warnings. But `merge_into_access_token_payload` calls `update_access_token_payload` so warning part is also difficult. I'll think about it and find something.